public/pages/CreateMonitor/components/DocumentLevelMonitorQueries/utils/helpers.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import _ from 'lodash';
+import { SUPPORTED_DOC_LEVEL_QUERY_OPERATORS } from './constants';
+import { OPERATORS_MAP } from '../../MonitorExpressions/expressions/utils/constants';
+
+/**
+ * Returns an array of { text: string, value: string, dataTypes: string[] } objects.
+ * Used by the DocumentLevelQuery.js to populate the operator options FormikSelect element.
+ * @return {*[]}
+ */
+export const getDocLevelQueryOperators = () => {
+  return SUPPORTED_DOC_LEVEL_QUERY_OPERATORS.map((type) => OPERATORS_MAP[_.toUpper(type)]);
+};
+
+/**
+ * Validates whether the array of document level monitor queries can be executed.
+ * @param queries - an array of FORMIK_INITIAL_DOCUMENT_LEVEL_QUERY_VALUES objects.
+ * @return {boolean} - TRUE if all queries are valid; otherwise FALSE.
+ */
+export const validDocLevelGraphQueries = (queries = []) => {
+  // The 'queryName', 'field', 'operator', and 'query' fields are required to execute a doc level query.
+  // If any of those fields are undefined for any queries, the monitor cannot be executed.
+  const incompleteQueries = queries.find(
```

Comment:
instead of `find` we should use `some` here and for return `!inCompleteQuery` will work.
Also, find will always return the first element which satisfies the condition, so it will not be queries but just query

Reply:
Ah, good suggestion! The way `find` is being used here seems to accomplish the same goal as `some`, but `some` is a more succinct way of accomplishing that goal. I'll revise this PR shortly.